Assets/MRTK/Core/Interfaces/InputSystem/IMixedRealityEyeGazeProvider.cs

```diff
@@ -50,6 +50,6 @@ public interface IMixedRealityEyeGazeProvider : IMixedRealityGazeProvider
         /// </summary>
         /// <param name="provider">The provider raising the event.</param>
         /// <param name="userIsEyeCalibrated">Boolean whether the user is eye calibrated or not.</param>
-        void UpdateEyeTrackingStatus(IMixedRealityEyeGazeDataProvider provider, bool userIsEyeCalibrated);
+        void UpdateEyeTrackingStatus(IMixedRealityEyeGazeDataProvider provider, bool? userIsEyeCalibrated);
```

Comment:
this is technically a breaking change. we should ensure it is documented in updating.md

Reply:
I knew there would be good steps I missed. =) Thanks for the guidance David!

Reply:
I don't see why this has to be a nullable variable.

Reply:
The same reason that bool? IsEyeCalibrationValid { get; } in the same interface is nullable.